### ISSUE TITLE
Problem in dataset

### ISSUE THREAD

**Issue by EnJiang** [kentsyx/pointnet-pytorch#6]
I think there is a problem within the dataset code for S3D.
The codes read one object in the Annotations folder as a sampling unit, which is wrong. In Annotations, each object is a fraction of a scene. For example, the object can be a widow while the scene is an office. Now during training, the codes sample batch_size number of **object**, rather than a scene. As a result, each sample's ground truth is just an array of the same numbers. E.g., if we sampled a window object and the label of a window is 7, the ground truth is an N-dimensional vector full of 7 where N is the number of points sampled.
I think the codes should sample scenes, not object.
Did I misunderstand something? Please let me know.
